docs/_book.yaml

```diff
@@ -27,92 +27,112 @@ upper_tabs:
-        path: /cirq/tutorials/basics
-      - title: "Textbook examples"
-        path: /cirq/tutorials/examples
-      - title: "Quantum variational algorithm"
-        path: /cirq/tutorials/variational_algorithm
-      - title: "QAOA experiment"
-        path: /cirq/tutorials/QAOA_Demo
-      - title: "Hidden linear function problem"
-        path: /cirq/tutorials/hidden_linear_function
-      - title: "Quantum walks"
-        path: /cirq/tutorials/Quantum_Walk
-      - title: "Rabi oscillation experiment"
-        path: /cirq/tutorials/Rabi_Demo
-
     - name: "Guide"
       contents:
+      ####  FIRST STEPS ####
+      - heading: "First steps"
+
       - title: "Install"
         path: /cirq/install
       ####  CONCEPTS ####
-      - heading: "Concepts"
-      - title: "Gates"
+      - heading: "Essential concepts"
+      - title: "Gates and operations"
         path: /cirq/gates
       - title: "Circuits"
         path: /cirq/circuits
       - title: "Simulation"
         path: /cirq/simulation
+
+      ####  FURTHER TOPICS ####
+      - heading: "Further topics"
       - title: "Noise"
         path: /cirq/noise
       - title: "Devices"
         path: /cirq/devices
-      - title: "Interoperability"
-        path: /cirq/interop
       - title: "Qudits"
         path: /cirq/qudits
-      ####  GOOGLE ####
-      - heading: "Google"
+      - title: "Import/export circuits"
+        path: /cirq/interop
+
+      ####  Contributor Guide ####
+      - heading: "Contributor guide"
+      - title: "Contribute"
+        path: /cirq/dev/development
+      - title: "RFC process"
+        path: /cirq/dev/rfc_process
+      - title: "Style guide"
+        path: /cirq/dev/style
+      - title: "Nomenclature"
+        path: /cirq/dev/nomenclature
+      - title: "Gate and operation guidelines"
+        path: /cirq/dev/gates
+      - title: "Plotting guidelines"
+        path: /cirq/dev/plotting
+      - title: "Serialization guidelines"
+        path: /cirq/dev/serialization
+
+      ####  GOOGLE HARDWARE ####
+      - heading: "Google hardware"
+      - title: "Concepts"
+        path: /cirq/google/concepts
+      - title: "Quantum Engine API"
+        path: /cirq/google/engine
       - title: "Google devices"
         path: /cirq/google/devices
-      - title: "Quantum Engine"
-        path: /cirq/google/engine
-      - title: "Device Specifications"
+      - title: "Device specifications"
         path: /cirq/google/specification
       - title: "Calibration metrics"
         path: /cirq/google/calibration
       - title: "Best practices"
         path: /cirq/google/best_practices
-      - title: "Reservations"
-        path: /cirq/tutorials/google/reservations
+
       ####  PASQAL ####
-      - heading: "Pasqal"
-      - title: "Quantum Circuits on Pasqal Devices"
-        path: /cirq/pasqal/getting_started
+      - heading: "Pasqal hardware"
       - title: "Pasqal devices"
         path: /cirq/pasqal/devices
       - title: "Pasqal sampler"
```

Comment:
Perhaps for another change, but I was initially confused what a "Pasqal sampler" was---is this a funny name for a demo/tutorial? (Looking at the doc I see this is not the case :)

Reply:
This is more of a reference as far as I'm concerned. Describes one important concept of the Pasqal API. Why do you think this is a demo/tutorial? It's under Guide.

Reply:
It was more the "sampler" name that threw me a bit. Not a big deal just caught my attention.

Reply:
Ah, I see, the technical term collides with the regular term "sampler" :D I am so used to it it didn't even occur to me. I'll think about alternate wording, thank you!